samples/MyLibrary/Class1.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Hosting.Internal;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Localization;
+
+namespace MyLibrary
+{
+    public class Class1
+    {
+        private IStringLocalizer<Class1> _localizer;
+
+        public Class1()
+        {
+            var services = new ServiceCollection();
```

Comment:
Would definitely like to see this using `IStringLocalizer<Class1>` via ctor injection to ensure the recommended end-to-end actually works.